plugins/woocommerce/includes/abstracts/abstract-wc-payment-gateway.php

```diff
@@ -206,7 +206,7 @@ public function get_method_description() {
 	 */
 	public function admin_options() {
 		$is_reactify_enabled      = Features::is_enabled( 'reactify-classic-payments-settings' );
-		$offline_payment_gateways = array( 'bacs', 'cheque', 'cod' );
+		$offline_payment_gateways = array( WC_Gateway_BACS::ID, \WC_Gateway_Cheque::ID, \WC_Gateway_COD::ID );
```

Comment:
This isn't wrong, of course, and it doesn't break any linting rules, but I'd personally think this is cleaner:

```suggestion
		$offline_payment_gateways = array( WC_Gateway_BACS::ID, WC_Gateway_Cheque::ID, WC_Gateway_COD::ID );
```

Just because we are already within the root namespace, so using `\` to qualify the classes doesn't add much (and it's a little odd to do it for just two of the referenced classes).

Reply:
Oh right. I removed all of them and imported the class when necessary on my latest commits.